modules/noise/doc_classes/Noise.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		This class defines the interface for noise generation libraries to inherit from.
-		A default get_seamless_noise() implementation is provided for libraries that do not provide seamless noise. This function requests a larger image from get_image(), reverses the quadrants of the image, then uses the strips of extra width to blend over the seams.
+		A default [code]get_seamless_noise()[/code] implementation is provided for libraries that do not provide seamless noise. This function requests a larger image from [code]get_image()[/code], reverses the quadrants of the image, then uses the strips of extra width to blend over the seams.
```

Comment:
I can't see the method `get_seamless_noise`, where is it defined?

Reply:
probably `[method get_seamless_image]` for 2d and `[method get_seamless_image_3d]` for 3d noise